spec/rails_helper.rb

```diff
@@ -1,12 +1,17 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
+
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../config/environment', __dir__)
+
 # Prevent database truncation if the environment is production
 abort 'Rails is running in production mode!' if Rails.env.production?
+
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+require 'support/factory_bot'
```

Comment:
Consider removing this line 13 and uncommenting  line 28 so all files under `spec/support` are required.